samza-api/src/main/java/org/apache/samza/application/StreamApplication.java

```diff
@@ -24,11 +24,46 @@
 
 
 /**
- * This interface defines a template for stream application that user will implement to create operator DAG in {@link StreamGraph}.
+ * This interface defines a template for stream application that user will implement to initialize operator DAG in {@link StreamGraph}.
```

Comment:
I always thought that the javadoc needs to refer to the readers as the third person. No strong opinion on this though. Only one thing that if we want to make this call, let's be consistent across the code base.